tests/CommandBusTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Telegram\Bot\Tests;
+
+use Prophecy\Argument;
+use Telegram\Bot\Objects\Update;
+use Telegram\Bot\Tests\Mocks\Mocker;
+use Telegram\Bot\Commands\CommandBus;
+use Telegram\Bot\Tests\Mocks\MockCommand;
+use Telegram\Bot\Tests\Mocks\MockCommandTwo;
+
+class CommandBusTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var CommandBus
+     */
+    protected $commandBus;
+
+    public function setUp()
+    {
+        $this->commandBus = new CommandBus(Mocker::createApi()->reveal());
+    }
+
+    public function testAddsAndReturnsInstantiatedCommands()
+    {
+        $this->commandBus->addCommand(MockCommand::class);
+        $commands = $this->commandBus->getCommands();
+        $this->assertInstanceOf(MockCommand::class, $commands['mycommand']);
+    }
+
+    public function testAddsMultipleCommands()
+    {
```

Comment:
This was done because I've copied those tests from `APITests` and that's how they were named there as they weren't "my" tests.

If we are happy with the underscore naming system then I'll update these.